pkg/apis/v1alpha1/subnetset_types.go

```diff
@@ -0,0 +1,78 @@
+/* Copyright © 2021 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: Apache-2.0 */
+
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type SubnetSetStatusCondition string
+
+const (
+	SubnetSetReady    SubnetSetStatusCondition = "Ready"
+	SubnetSetNotReady SubnetSetStatusCondition = "NotReady"
+)
+
+// SubnetSetSpec defines the desired state of SubnetSet
+type SubnetSetSpec struct {
+	// Size of subnet based upon estimated workload count.
+	// Defaults to 64.
+	// +kubebuilder:default:=64
+	IPV4SubnetSize int `json:"ipv4SubnetSize,omitempty"`
+	// Access mode of subnet, accessible only from within VPC or from outside VPC.
+	// Defaults to private.
+	// +kubebuilder:default:=private
+	// +kubebuilder:validation:Enum=private;public;isolated
+	AccessMode string `json:"accessMode,omitempty"`
+}
+
+// SubnetSetCondition defines condition of SubnetSet.
+type SubnetSetCondition struct {
+	// Type defines condition type.
+	Type SubnetSetStatusCondition `json:"type"`
+	// Status defines status of condition type, True or False.
+	Status corev1.ConditionStatus `json:"status"`
+	// Reason shows a brief reason of condition.
+	Reason string `json:"reason,omitempty"`
+	// Message shows a human readable message about condition.
+	Message string `json:"message,omitempty"`
+}
+
+// SubnetItem defines subnet items of SubnetSet.
```

Comment:
subnet -> Subnet

Reply:
defines the observed state of a single Subnet of a SubnetSet

Reply:
Thanks and fixed.